session4/src/com/itschool/homework/Challenge5.java

```diff
@@ -0,0 +1,36 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+public class Challenge5 {
+
+    public static void main(String[] args) {
+//
+//        int x = 10;
+//        int y = 11;
+//        //assignment and compound assignment operators
+//
+//        System.out.println(x*=5);
+//        System.out.println(x%=3);
+//        System.out.println(y%=3);
+//        System.out.println(x+=11);
+//        System.out.println(x-=1);
+//        System.out.println(x/=2);
+
+        Scanner input = new Scanner(System.in);
+
+        int price = 10;
+        int age = 56;
+        double payAmount = 0;
+
+        if(age >+ 60){
+
+            payAmount = price - price * 5 / 100;
+
+        } else if(age >= 55){
```

Comment:
the discount applies depending on age? Nice you are using `if-else` block even tough we didn't study it yet :)
However, what happens if the my age is under 55? I don't get a discount? :(
PS: if you don't care about the age (it is not requested in the challange), you could use the already studied **ternary operator** here.

Reply:
sure, I know we didn't practice the ternary operator so far, we will do it in the today's course.
In the meantime please commit all the new changes/improvements you made so I can seem them here in GitHub.